### PR TITLE
update FAQ and post a blog about content types

### PR DESCRIPTION
**Pull request by hannahfrost**
Also add DPLAfest presentation to upcoming presentations list

### CODE REVIEW

**Comment by mjgiarlo** [comment]
:+1:

File: _data/upcoming_presentations.yml [hunk 1]
```diff
@@ -1 +1,8 @@
-- title: Nothing scheduled at this time.
+- title: Hydra in a Box - Building a Next-Generation Platform for Digital Collections
+  presenters:
+    - Mark Matienzo
+    - Gretchen Gueguen
```
Comment (anarchivist) [inline _data/upcoming_presentations.yml]: @hannahfrost Are you going to present with us? (Do you want to?)

File: _posts/2016-03-04-content_types_news.markdown [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+#### Field Notes
+Our environmental scan and design research shows that image-based content tops the list of content types managed in digital repositories. More surprising is the fact that images top the list of content types currently _not_ managed in repositories (due to variety of limiting factors, such as insufficient staff or other resources), according to our survey results. Cultural heritage institutions with digital collections are brimming with images and are looking for a modern, full-featured repository application that can take advantage of the latest technologies – including [IIIF]( http://iiif.io/) -- for managing, making discoverable and delivering image assets online. Hydra-in-a-Box is well positioned to meet this need.
```
Comment (anarchivist) [inline _posts/2016-03-04-content_types_news.markdown]: Extra space in the link

**Comment by anarchivist** [comment]
Looks good but I've added a minor change and a minor question.

**Comment by anarchivist** [comment]
How is this handled by Rails? I guess I don't have a strong opinion about it, but it would be nice if the language were consistent.

File: _data/upcoming_presentations.yml (hunk 1 above)
Comment (hannahfrost) [inline _data/upcoming_presentations.yml]: Oh sure, @anarchivist. I'd be delighted! I'll add my name. Thanks.

**Comment by hannahfrost** [comment]
Made these minor changes, so merging now.

**Comment by mjgiarlo** [comment]
:clap:

**Pull request merged**